ee/vellum_ee/workflows/display/nodes/vellum/retry_node.py

```diff
@@ -39,4 +47,86 @@ def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> J
             "attributes": attributes,
         }
 
-        return super().serialize(display_context, adornment=adornment)
+        serialized_node = super().serialize(
+            display_context,
+            adornment=adornment,
+        )
+
+        if serialized_node["type"] == "GENERIC":
+            return serialized_node
+
+        serialized_node_definition = serialized_node.get("definition")
+        if isinstance(serialized_node_definition, dict):
+            serialized_node_definition_module = serialized_node_definition.get("module")
+            if isinstance(serialized_node_definition_module, list):
+                serialized_node_definition_module.extend(
+                    [
+                        serialized_node_definition["name"],
+                        ADORNMENT_MODULE_NAME,
+                    ]
+                )
+                serialized_node_definition["name"] = node.__name__
+
+        return serialized_node
+
+    def get_node_output_display(self, output: OutputReference) -> Tuple[Type[BaseNode], NodeOutputDisplay]:
+        inner_node = self._node.__wrapped_node__
+        if not inner_node:
+            return super().get_node_output_display(output)
+
+        node_display_class = get_node_display_class(BaseNodeDisplay, inner_node)
+        node_display = node_display_class()
+
+        inner_output = getattr(inner_node.Outputs, output.name)
+        return node_display.get_node_output_display(inner_output)
+
+    @classmethod
+    def wrap(
```

Comment:
Didn't review this closely since you closed the PR, but you're right in that we'll need some version of this before we publish 0.14.11